src/components/homePage/Speakers.vue

```diff
@@ -1,13 +1,36 @@
 <template>
 <div id="speakers">
 	<h2>CUTC 2020</h2>
-	<!-- TAB LAYOUT -->
+	<div id="tabs">
+		<p id="tab" @click="currentPage = 0"> Speakers </p>
+		<p id="tab" @click="currentPage = 1"> Events </p>
+	</div>
+	<vueper-slides v-if="currentPage == 0">
+		<vueper-slide v-for="i in 5" :key="i" :title="i.toString()" />
+	</vueper-slides>
+	<vueper-slides v-else>
+		<vueper-slide v-for="i in 5" :key="i" :title="i.toString()" />
+	</vueper-slides>
 </div>
 </template>
 
 <script>
+import { VueperSlides, VueperSlide } from 'vueperslides'
+import 'vueperslides/dist/vueperslides.css'
+
 export default {
+	components: { VueperSlides, VueperSlide },
 	name: 'Speakers',
+	data: () => ({
+		slides: [
+			{
+				title: 'Slide #1',
+				content: 'Slide content.'
+				}
+				],
```

Comment:
I know we didn't talk about this, but it'd be great if the data was pulled from a JSON in the content folder, similar to adib or sanaa's PRs.